src/ending/podium_ceremony_actors.c

```diff
@@ -137,35 +137,39 @@ void clear_D_802874D8_actors() {
 }
 
 u16 random_u16_credits(void) {
-    u16 temp1, temp2;
+    if (CVarGetInteger("gBugfixRNGReset", true) == false) {
```

Comment:
I don't quite understand this change.

The idea is to remove the duplicate function and use the original one. But now the random seed won't have been reset because we're now using gRandomSeed instead of sRandomSeed.


I would recommend one of two options
1) Just delete random_u16_credits, and modify the game to act how expected using just the one random function
* This is simpler, makes more sense, etc.

2) Not call the original `random_u16()` and just use random_u16_credits()
* Otherwise it's more likely to get mixed up and confused.